src/m365/spo/commands/file/file-retentionlabel-ensure.ts

```diff
@@ -92,7 +98,16 @@ class SpoFileRetentionLabelEnsureCommand extends SpoCommand {
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     try {
-      const fileProperties = await this.getFileProperties(logger, args);
+      const fileProperties: FileProperties = await this.getFileProperties(logger, args);
+
+      const labelInformation: ListItemRetentionLabel = await spo.getWebRetentionLabelInformationByName(args.options.webUrl, args.options.name);
```

Comment:
Hi @nicodecleyre, I'm seeing this still in here though. That should have been rolled back as well.